playground/src/components/editor/index.js

```diff
@@ -0,0 +1,53 @@
+
+/**
+ * WordPress dependencies
+ */
+import '@wordpress/editor'; // This shouldn't be necessary
```

Comment:
Why is it? If it's intended to be merged this way, can the comment be updated to detail why it exists and what's necessary to see it removed (or a link to an equivalent issue)?